BookingsApi/BookingsApi.IntegrationTests/Database/Commands/AddParticipantsToVideoHearingCommandTests.cs

```diff
@@ -262,9 +262,8 @@ public async Task Should_Add_New_Participant_Linked_To_Existing_Participant()
             returnedVideoHearing.Participants.Where(x => x.LinkedParticipants.Any()).Should().NotBeNull();
         }
 
-        [TestCase("Judge", "Judge")]
-        [TestCase("Staff Member", "Staff Member")]
-        public async Task Should_add_participant_to_video_hearing(string caseRole, string hearingRole)
+        [Test]
+        public async Task Should_add_participant_to_video_hearing()
```

Comment:
could we rename this to `Should_add_judge_participant_to_video_hearing()`, if it only adds the judge now - makes it clearer which type of participant it is adding

Reply:
done